### DISCUSSION TITLE
How can I load animation file to model dynamicly

### DISCUSSION THREAD

**Discussion by JLTG2018** [google/filament#6209] [General]
I want to load animation file individuly, and apply this animation to my bot model.How can I do this.

I have a lot of animation file, and I want to change my bot animation dynamicly.

**Comment by JLTG2018** [comment]
[glb.zip](https://github.com/google/filament/files/9808745/glb.zip)

**Reply by romainguy** [reply]
glTF files can contain multiple animations and our APIs provide a way to select which animation you want to play. You can see an example here: https://github.com/google/filament/blob/main/android/samples/sample-gltf-viewer/src/main/java/com/google/android/filament/gltf/MainActivity.kt#L388

**Reply by ritik619** [reply]
@romain guy 
May i know is it possible to load a animations from a seperate glb file? And apply it on the existing  model?
Like we do in ThreeJs?

**Reply by ritik619** [reply]
From what i Infer the above example is when there are Animations already in the glb of Avatar/model

**Reply by ritik619** [reply]
@JLTG2018 were you able to figure out any way?

**Reply by ninjz** [reply]
Anyone have this figured out? Seems to be a big limitation from using this library. I am able to get this working on ThreeJS and SceneKit on iOS.
